js/compiler_test.ts

```diff
@@ -177,7 +177,7 @@ function globalEvalMock(x: string): void {
 function logMock(...args: any[]): void {
   logStack.push(args);
 }
-const osMock: compiler.Os = {
+const osMock = {
```

Comment:
These interfaces (Os, Ts) are now `@internal` because they leak stuff we don't want into the global types therefore they aren't visible during the unit tests which run in userland.